src/Server/WishList.fs

```diff
@@ -69,6 +76,7 @@ let postWishList (ctx: HttpContext) =
                     return! Successful.OK (JsonConvert.SerializeObject wishList) ctx
                 else
                     return! BAD_REQUEST "WishList is not valid" ctx
-        with
-        | _ -> return! SERVICE_UNAVAILABLE "Database not available" ctx
+        with exn -> 
+            printfn "SERVICE_UNAVAILABLE, %A" exn
```

Comment:
Feels like we should always put ``printfn`` or some other logging in catastrophic server-side error cases